config/routes.rb

```diff
@@ -18,6 +18,7 @@
   namespace :citizens do
     resources :legal_aid_applications, only: [:show]
     resource :consent, only: %i[show create]
+    resource :property_value, only: %i[show create]
```

Comment:
The pattern being used in the providers journey is show and update. So I think the form should be submitted to `update` rather than `create`. Not that the form method will need to change to PATCH to match.